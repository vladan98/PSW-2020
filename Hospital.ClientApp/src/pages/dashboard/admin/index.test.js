import AdminDashboard from "."
import render from "../../../helpers/renderComponent"

describe("AdminDashboard", () => {
    it("should render admin dashboard", () => {
        const { container } = render(<AdminDashboard />)

        expect(container).toMatchSnapshot()
    })
})