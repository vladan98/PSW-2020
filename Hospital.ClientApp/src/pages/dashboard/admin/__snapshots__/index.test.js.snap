// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminDashboard should render admin dashboard 1`] = `
<div>
  <main
    class="MuiContainer-root MuiContainer-maxWidthXs"
  >
    <div
      class="makeStyles-paper-1"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          class="MuiPaper-root MuiCard-root makeStyles-card-2 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <div
              class="MuiCardContent-root"
            >
              <h2
                class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
              >
                View Malicious patients
              </h2>
            </div>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiPaper-root MuiCard-root makeStyles-card-2 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <button
            class="MuiButtonBase-root MuiCardActionArea-root"
            tabindex="0"
            type="button"
          >
            <div
              class="MuiCardContent-root"
            >
              <h2
                class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom"
              >
                View feedback
              </h2>
            </div>
            <span
              class="MuiCardActionArea-focusHighlight"
            />
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </main>
</div>
`;
